lib/esm/serialize.js

```diff
@@ -0,0 +1,166 @@
+import { integers } from './types';
+import { EncodeBuffer } from './buffer';
+import BN from 'bn.js';
```

Comment:
Same problem for bn. Is bn lib necessary? Can we simply use [JS native BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)?